style/base.css

```diff
@@ -143,3 +148,20 @@
   border-radius: var(--je-round-corners);
   background: transparent;
 }
+
+.jp-ToolbarButtonComponent.jp-mod-jeCreateCopyButton {
+  background: var(--je-slate-blue);
+  color: white;
+  border-radius: var(--je-round-corners);
+  font-family: var(--je-font-family);
+  font-weight: 600;
+  text-transform: none;
+  padding: 4px 12px;
+  border: none;
+  box-shadow: 1px 2px 0 #0000006f !important; /* pretty much convinced my browser is haunted */
+}
+
+.jp-ToolbarButtonComponent.jp-mod-jeCreateCopyButton:hover {
+  background: #2c1f5c;
+  color: white;
```

Comment:
That's the same as for non-hover state, right?

```suggestion
```

Reply:
You are right, this is something I added when debugging, but forgot to drop later. Thanks for the catch!